src/gui/qgsfilewidget.h

```diff
@@ -184,6 +184,9 @@ class GUI_EXPORT QgsFileWidget : public QWidget
     //! emitted as soon as the current file or directory is changed
     void fileChanged( const QString & );
 
+    //! emitted before and after showing the file dialog
+    void blockEvents( bool );
```

Comment:
```suggestion
    void blockEvents( bool ) SIP_SKIP;
```

This will omit the signal from the PyQt bindings, which is desirable because it's just an internal implementation detail